cmd/kubeadm/app/util/config/masterconfig.go

```diff
@@ -97,6 +98,14 @@ func SetNodeRegistrationDynamicDefaults(cfg *kubeadmapi.NodeRegistrationOptions,
 		cfg.Taints = []v1.Taint{kubeadmconstants.MasterTaint}
 	}
 
+	if cfg.CRISocket == "" {
+		cfg.CRISocket, err = kubeadmruntime.DetectCRISocket()
+		if err != nil {
+			return err
+		}
+		glog.V(1).Infof("detected and using CRI socket: %s\n", cfg.CRISocket)
```

Comment:
- glog calls don't need the extra `\n`. appears multiple times.
- better move the Infof() call in DetectCRISocket()?
- shouldn't we add this for nodeconfig.go too?
JoinConfiguration has a NodeRegistrationOptions too.